canonical-web-and-design/ubuntu.com
# Reactify remainder of microcerts page

## Done

- Reactify the remainder of the microcerts page
  - Microcerts table which is displayed when all certs are complete
  - Footer that shows different content based on whether the user is enrolled

## QA

- Run the server with `dotrun`
- Go to http://localhost:8001/cube/microcerts?test_backend=true
- Verify that the footer is displayed. (If you have enrollments, it should thank you for joining the beta. If you have no enrollments, there should be a message a button allowing you to join the beta.)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/commercial-squad/issues/424

## Screenshots

### Footer (has enrollments)
![microcerts-footer-enrolled](https://user-images.githubusercontent.com/995051/146372536-2ed07c52-a9ef-42c0-a66e-f0aa22f37ad2.png)

### Footer (no enrollments)
![microcerts-footer-apply](https://user-images.githubusercontent.com/995051/146372569-1247de94-22f7-45e3-aef5-579da7d07e13.png)
